validator/errorbundler.py

```diff
@@ -74,6 +72,8 @@ def __init__(self, determined=True, listed=True, instant=False,
 
         self.supported_versions = self.for_appversions = for_appversions
 
+    SEVERITIES = {"trivial", "low", "medium", "high"}
```

Comment:
I don't remember seeing anything about `trivial` in your notes: I remember that a low severity would still let an addon be automatically reviewed (pass validation) if it had the exact same warning in a previously reviewed version. How about `trivial`?